### PR TITLE
Cpu sys call

### PR DESCRIPTION
**Pull request by tgujar**
added sys call measurement

### CODE REVIEW

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (acbokade) [inline .vscode/c_cpp_properties.json:1]: Can we remove this by adding in gitignore?

Comment (tgujar) [reply .vscode/c_cpp_properties.json:1]: we need this so vscode doesn't throw error squiggles

Comment (tgujar) [reply .vscode/c_cpp_properties.json:1]: if you are using another editor feel free to ignore, I don't think it affects stuff

File: src/CPU/syscall.cpp [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+#include "proc.hpp"
+
+
+using namespace std;
+
+// getpid() cache has been deprecated, see https://man7.org/linux/man-pages/man2/getpid.2.html
+
+double sys_call_time_chrono() {
+    auto start = chrono::steady_clock::now();
+    for(ssize_t i = 100000; i > 0; i--) {
```
Comment (acbokade) [inline src/CPU/syscall.cpp:17]: Can we define this as a constant?

**Review by acbokade — APPROVED** [review]

File: src/CPU/syscall.cpp (hunk 2 above)
Comment (tgujar) [reply src/CPU/syscall.cpp:17]: yeah sure, any large number works. I don't think there is any reason to change the value, so I didn't define it separately

**Pull request merged**